### ISSUE TITLE
Search web for the selected celestial object and share links

### ISSUE THREAD

**Issue by salarmehr** [Stellarium/stellarium#3256]
Could we have a link to a search engine to search the web for the selected celestial objects? (that opens the default web browser). 
Additionally, we can have a share button so people be able to see the selected object in their app like what happens when you click on a location link and it opens google Maps. In Stellarium case, the link should have the selected timestamp included as well.

**Comment by alex-w** [comment]
Please use Online Queries plugin

**Comment by gzotti** [comment]
Yes, you should be able to configure an URL in OnlineQueries (Guide, ch. 14.8).
I do not understand the scenario about the share button, please elaborate a bit more.

**Comment by gzotti** [comment]
Solution in config.ini to use Google with OnlineQueries configuration:

```
[OnlineQueries]
custom2_url   = https://www.google.com/search?q=%1
custom2_use_hip   = false
```

**Comment by salarmehr** [comment]
@gzotti  The concept of a "Share" button in Stellarium is a valuable addition to enhance user experience and share astronomical observations seamlessly with others. Here's a breakdown of how this idea could work:

1. **User Interaction:** Within the Stellarium application, users can interact with the celestial objects and select a specific object, such as a star, planet, or constellation.

2. **Share Button:** To share the selected object, a "Share" button can be incorporated into the user interface. This button is easily accessible, either as an icon or a menu option, making it intuitive for users to locate.

3. **Generating a Shareable Link:** When the user clicks the "Share" button, the application generates a shareable link that encapsulates information about the selected object and any relevant data, such as the timestamp of the observation.

4. **Timestamp Inclusion:** As per the request, the selected timestamp should be included in the generated link. This timestamp would reflect the exact moment when the user selected the object. It provides context to the shared observation, allowing others to understand what was visible in the night sky at that specific time.

5. **Sharing Options:** After generating the shareable link, users can choose from various sharing options. These may include sharing via email, messaging apps, social media, or even copying the link to the clipboard for manual sharing.

6. **Recipient Interaction:** When someone receives the shared link, they can click on it, leading them to a dedicated Stellarium web page or a mobile app screen. This page/screen will display the selected object and the associated timestamp.

7. **Compatibility:** To ensure a smooth user experience, the shared link should automatically open Stellarium (if installed) or provide an option to install or open the application if it's not already installed on the recipient's device.

8. **Enhanced Engagement:** This feature not only facilitates sharing but also encourages engagement with the Stellarium app among a broader audience. Recipients of the shared link may be intrigued to explore the night sky themselves, thereby expanding the user base.

In summary, the addition of a "Share" button in Stellarium, with the selected timestamp included, would provide a convenient way for users to share their celestial observations and promote engagement with the application. This feature not only enhances user interaction but also encourages the exploration and appreciation of the night sky among a wider audience.

**Comment by gzotti** [comment]
OK. I leave it for whoever wants to develop a share button. I don't see the scenario to receive a link via email or even smartphone SM app that invites me to install a 600MB application on my desktop computer, transfer the URL from smartphone to desktop, and rejoyce for a moment over a single view. This may be better suited for Stellarium-web. 

For Stellarium desktop, you can create an Observing List, export it and send around.

**Comment by salarmehr** [comment]
@gzotti you usually share it with other night sky enthusiast people that you know have the app installed.

**Comment by gzotti** [comment]
The existing feature in the desktop program that lets you store, export or import your remarkable objects or preconfigured views and share (via external means) with others who have the program installed is called Observing Lists. They will work better in the upcoming 23.3 (try current beta!). I seriously do not see the need to invest a week or so of my own spare time to add a parallel feature for this sub-purpose of them. I'd rather go out observing.
